RadarSDK/RadarTrackingOptions.m

```diff
@@ -219,8 +239,8 @@ - (NSDictionary *)dictionaryValue {
     dict[kDesiredAccuracy] = [RadarTrackingOptions stringForDesiredAccuracy:self.desiredAccuracy];
     dict[kStopDuration] = @(self.stopDuration);
     dict[kStopDistance] = @(self.stopDistance);
-    dict[kStartTrackingAfter] = self.startTrackingAfter;
-    dict[kStopTrackingAfter] = self.stopTrackingAfter;
+    dict[kStartTrackingAfter] = self.startTrackingAfter ? @([self.startTrackingAfter timeIntervalSince1970] * 1000) : nil;
+    dict[kStopTrackingAfter] = self.stopTrackingAfter ? @([self.stopTrackingAfter timeIntervalSince1970] * 1000) : nil;
```

Comment:
These two lines cause the `XCTAssertEqualObjects()` line in the `RadarSDKTests.test_Radar_startTracking_custom()` test to fail, because now it's trying to compare an `NSDate` with a `TimeInterval`. The test passes if these lines are reverted.